src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs

```diff
@@ -52,5 +53,12 @@ public async Task<IActionResult> AddEmployerVendorId([FromRoute] string hashedLe
             await SendCommandAsync(new AddEmployerVendorIdForLegalEntityCommand(hashedLegalEntityId, request.EmployerVendorId));
             return NoContent();
         }
+
+        [HttpPatch("/legalentities/refresh")]
```

Comment:
This already exists as a Job resource so not needed as an RPC style interface here.

Reply:
The Job command endpoint uses a Dictionary<string,string> which isn't suitable for what's needed - the outer API now passes in the set of ALEs fetched from the accounts API and the paging info.

Reply:
It uses a Dictionary<string,object> so any json object could be passed to it.

Reply:
Alternatively you could create a "LegalEntitiesRefreshJobs/ endpoint and POST a LegalEntitiesRefreshJob resource to that.  As the technical requirement is to produce a REST API the URL should point to a resource (which would be a noun not a verb).   Refresh isn't a resource of a LegalEntity.